otherlibs/unix/unix.mli

```diff
@@ -854,18 +855,19 @@ val mkfifo : string -> file_perm -> unit
 val create_process :
   string -> string array -> file_descr -> file_descr ->
     file_descr -> int
-(** [create_process prog args stdin stdout stderr]
-   creates a new process that executes the program
-   in file [prog], with arguments [args]. The pid of the new
-   process is returned immediately; the new process executes
-   concurrently with the current process.
-   The standard input and outputs of the new process are connected
-   to the descriptors [stdin], [stdout] and [stderr].
-   Passing e.g. {!Unix.stdout} for [stdout] prevents the redirection
-   and causes the new process to have the same standard output
-   as the current process.
-   The executable file [prog] is searched in the path.
-   The new process has the same environment as the current process. *)
+(** [create_process prog args stdin stdout stderr] creates a new process
+    that executes the program in file [prog], with arguments [args]. Note that
+    the first argument, [args.(0)], is by convention the filename of the
+    program being executed, just like [Sys.argv.(0)].
+
+    The pid of the new process is returned immediately; the new process
+    executes concurrently with the current process.  The standard input and
```

Comment:
If you are separating paragraphs by a newline, you should do so here too (and in the rest of the paragraph below).